docs/pages/reference/join-methods.mdx

```diff
@@ -440,6 +440,19 @@ Support for self-hosted Terraform Enterprise requires Teleport Enterprise.
 - [Run the Teleport Terraform Provider on Terraform Cloud](../admin-guides/infrastructure-as-code/terraform-provider/terraform-cloud.mdx)
 </Admonition>
 
+### Spacelift: `spacelift`
+
+This join method is used to authenticate using Spacelift. It is typically used
+by the Teleport Terraform provider on either Spacelift cloud or self-hosted
+Spacelift. It can not be used to join Terraform runs on other platforms and
+dedicated join methods should be used instead.
```

Comment:
> It can not be used to join Terraform runs on other platforms and dedicated join methods should be used instead.

I would structure this sentence differently since the conjunction of the descriptive clause and the "should" clause reads a little awkwardly to me. Maybe use ", so" instead of "and", or use two separate sentences.

Reply:
8b49c51827